ludwig/distributed/base.py

```diff
@@ -171,9 +171,6 @@ def prepare_before_load(self) -> bool:
     def is_model_parallel(cls) -> bool:
         return False
 
-    def eval(self, model: nn.Module):
-        model.eval()
```

Comment:
Is the reason we're getting rid of this over here because we're just going to be running this through the Predictor classes now, which will change model to eval mode at that point?

In that case, would it not make sense to still do something like self.model.eval()? Although I guess yeah, you don't need a function for this

Reply:
We removed this because DeepSpeed fixed the issue that required it. Now the eval flag is set directly instead of going through the distributed backend.